app/src/main/java/by/academy/jc/klimakhovich_ht3/FrogJumper.java

```diff
@@ -0,0 +1,21 @@
+package by.academy.jc.klimakhovich_ht3;
+
+public class FrogJumper {
+    public int frog1(int x, int y, int d) {
```

Comment:
Think about better solution that would work faster. You probably don't need to use cycles at all.